src/style.css

```diff
@@ -1,3 +1,70 @@
+:root {
+  --light-blue: #00acdc;
+  --black: #000;
```

Comment:
Better not to use color names in variable name
Better to use name like '--main-color'...